src/core/server/rendering/rendering_service.tsx

```diff
@@ -101,6 +101,12 @@ export class RenderingService {
             uiSettings.getOverrideOrDefault('theme:darkMode')) ||
           false;
 
+        // At the very least, the schema should define a default theme; the '' will be unreachable
+        const themeVersion =
+          (settings.user?.['theme:version']?.userValue ??
+            uiSettings.getOverrideOrDefault('theme:version')) ||
+          '';
```

Comment:
why do we need the fallback if it really is unreachable?

Reply:
If for whatever reason it lands with something falsy, at least it will look nice in the error.